server/item/enchantment/curse_of_binding.go

```diff
@@ -0,0 +1,51 @@
+package enchantment
+
+import (
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+)
+
+// CurseOfBinding is an enchantment that prevents the removal of an equipped item enchanted with the curse of binding from its armour slot.
+type CurseOfBinding struct{}
```

Comment:
The naming of the file (`curse_of_binding.go`) is not consistent with the naming of Curse of Vanishing (`vanishing.go`)

Reply:
I would argue that the naming of vanishing.go is not consistent with the naming of all the other enchantment files, which use the full enchantment names, with spaces represented as underscores.

Reply:
Fair enough. Feel free to change `vanishing.go`.